test/org/sagebionetworks/bridge/services/HealthCodeServiceTest.java

```diff
@@ -24,6 +34,16 @@
     @Resource
     private HealthCodeService healthCodeService;
 
+    @Before
+    public void before() {
+        clearDynamo();
+    }
+
+    @After
+    public void after() {
+        clearDynamo();
+    }
```

Comment:
removed (this must have been removed by your consent services changes, then added back)